core-tests/jvm/src/test/scala/zio/SerializableSpec.scala

```diff
@@ -128,8 +151,8 @@ object SerializableSpec extends ZIOBaseSpec {
     },
     testSync("FiberFailure is serializable") {
       val failure = FiberFailure(Cause.fail("Uh oh"))
-      assert(serializeAndDeserialize(failure))(equalTo(failure))
-    },
+      assert(serializeAndDeserialize(failure))(equalToIgnoringStackTrace(failure))
```

Comment:
The failure in the JVM test is due to the comparison of FiberFailure instances which now includes the javaStackTrace. The deserialization process may not preserve the exact identity of the `StackTraceElement` array, causing the equality check to fail.

To fix this, I have introduced custom the equality check for `FiberFailure` to ignore the `javaStackTrace` field in the comparison and focusing on the important parts of FiberFailure while ignoring the potentially varying details of the javaStackTrace.